### PR TITLE
Derive hasReassignments from Deliveries

### PR DESCRIPTION
**Pull request by charlie-tf**
- information is in the deliveries, instead of needing an additional query
- hasReassignments is true if any of the delivery types is
  "Reassignment To" or "Reassignment From"
- also add back in the query for the destination planting site name

### CODE REVIEW

File: src/components/NurseryWithdrawals/WithdrawalDetails/WithdrawalTabPanelContent.tsx [hunk 1]
```diff
@@ -41,19 +53,23 @@ export default function WithdrawalTabPanelContent({
...
     },
     {
       title: strings.DESTINATION,
-      data: withdrawalSummary?.destinationName ?? '',
+      data: siteName,
     },
     {
       title: strings.TO_PLOT,
-      data: withdrawalSummary?.plotNames ?? '',
+      data:
```
Comment (constanzauanini) [inline src/components/NurseryWithdrawals/WithdrawalDetails/WithdrawalTabPanelContent.tsx:68]: We were showing the plot name before and now just the id, is that intentional ?

Comment (charlie-tf) [reply src/components/NurseryWithdrawals/WithdrawalDetails/WithdrawalTabPanelContent.tsx:68]: Yeah, the search result was nice in that it had a lot of this information populated in the summary. I will take a look at the `getPlantingSiteData()` method though as well.

**Comment by charlie-tf** [comment]
Going to leave in the search query for now. I'll create a new PR to show/hide the Reassign button.

**Comment by karthikbtf** [comment]
@charlie-tf you would still need to hide the reassign button for deliveries that don't involve a plot.
You can use the existence of plotNames as the criterion.

**Pull request closed without merging**